testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/jsf/unsupportedwar/HelloBean.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2020 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jboss.as.test.integration.jsf.unsupportedwar;
+
+import java.io.Serializable;
+import javax.enterprise.context.RequestScoped;
+import javax.inject.Named;
+
+@Named(value = "helloBean")
+@RequestScoped
+public class HelloBean implements Serializable {
```

Comment:
Harmless, but Serializable is not needed for RequestScope, feel free to remove it to make the code more clean.